src/psyclone/tests/psyir/nodes/operation_test.py

```diff
@@ -114,6 +136,7 @@ def test_binaryoperation_create():
     rhs = Reference(DataSymbol("tmp2", REAL_SINGLE_TYPE))
     oper = BinaryOperation.Operator.ADD
     binaryoperation = BinaryOperation.create(oper, lhs, rhs)
+    assert not binaryoperation._has_explicit_grouping
```

Comment:
Please add a check that `has_explicit_grouping` can be passed to the `create()` method.

Reply:
Done